src/explore-education-statistics-frontend/src/pages/_app.tsx

```diff
@@ -31,12 +38,16 @@ class App extends BaseApp {
   }
 
   public render() {
-    const { Component, pageProps } = this.props;
+    // @ts-ignore
+    const { Component, pageProps, cookieBannerProps } = this.props;
 
     return (
       <Container>
         <Helmet titleTemplate="%s - GOV.UK" />
-        <Component {...pageProps} />
+        <CookiesProvider>
```

Comment:
Looking at the docs for `react-cookies`, it looks like we can actually pipe through the cookies from the server here. Think this would avoid the whole `CookieBanner.getInitialProps` thing?

See: https://github.com/reactivestack/cookies/tree/master/packages/react-cookie/#server-rendering-example